packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.routing.js

```diff
@@ -172,11 +172,19 @@ export default /* @ngInject */ ($stateProvider) => {
               return getNodes(database).then((nodes) =>
                 database.setNodes(nodes),
               );
-            })
-            .finally(() => {
-              return pollNodesStatus();
-            });
+            },
+            null,
+            () => {
```

Comment:
Could you keep the form `then` / `catch` / `finally` please?

Reply:
Hi @JeremyDec ,
Here i'm using the `ng-ovh-swimming-poll`, which can take 3 functions as parameters in the `then` to handle the success, error, and pending states.
See documentation here: https://www.npmjs.com/package/@ovh-ux/ng-ovh-swimming-poll